packages/cluster-ui/src/statementsTable/statementsTable.tsx

```diff
@@ -31,6 +32,7 @@ const longToInt = (d: number | Long) => Number(FixLong(d));
 
 function makeCommonColumns(
   statements: AggregateStatistics[],
+  totalWorkload?: number,
```

Comment:
Is it correct to make this optional? When is it not specified?

The other problem with making it optional is that I think if it's not specified that's an almost guaranteed divide by 0 exception below, right?

Reply:
good point! Fixed